Bedrock-OSS/bedrock-wiki-vite
# Fix incorrect emoji name

# Summary
Change the name of emoji `U+E057` from "Small Up Down"  to "Small Down Arrow" to resolve a typo.
Closes #643  
# Contributions:
@ThomasOrs 

# To Test:
- Go to `/concepts/emojis.html#mobile-ios-android`
- Confirm the emoji below `Small Up Arrow` is `Small Down Arrow`
